simconnect-sdk/src/domain/client_event.rs

```diff
@@ -126,6 +162,40 @@ pub enum ClientEvent {
     },
     /// Toggles parking brake on/off.
     ParkingBrakes,
+    // Comm Radios
+    Com1RadioStbySet {
+        value: i32,
```

Comment:
I don't think that BCD16 values can be negative, so probably this sits better as `u32`.